content/intro-to-storybook/react/ja/get-started.md

```diff
@@ -11,30 +11,25 @@ Storybook は開発時にアプリケーションと並行して動きます。S
 
 ## React 向けの Storybook を構築する
 
-Storybook を開発プロセスに組み込むにあたり、いくつかの手順を踏む必要があります。まずは、[Create React App](https://github.com/facebook/create-react-app) (CRA) を使用してアプリケーションを作成し、[Storybook](https://storybook.js.org/) とテストフレームワークの [Jest](https://facebook.github.io/jest/) を有効にしましょう。それでは、次のコマンドを実行してください:
+Storybook を開発プロセスに組み込むにあたり、いくつかの手順を踏む必要があります。まずは、[degit](https://github.com/Rich-Harris/degit) を使用してビルド環境をセットアップしましょう。このパッケージを利用するにあたって、テンプレート（アプリケーションの一部をデフォルト設定で構築したもの）が開発ワークフローの短縮に役立ちます。
+
+それでは、次のコマンドを実行してください:
 
 ```bash
-# アプリケーションを作成する:
-npx create-react-app taskbox
+# テンプレートをクローンする:
+npx degit chromaui/intro-storybook-react-template taskbox
 
 cd taskbox
 
-# Storybook を追加する:
-npx -p @storybook/cli sb init
+# 依存パッケージをインストールする:
+yarn
 ```
```

Comment:
@bhbs if you could revert the comments in this section back to English, I'd appreciate it. We're aware that translating into a native language brings the translation closer to home (pardon the bad pun), but until we have a better way of handling it we prefer to keep them as so.

Reply:
Reverted 👍 6edfb80